specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/FlexibleServers.json

```diff
@@ -361,6 +361,51 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBforPostgreSQL/flexibleServers/{serverName}/replicas": {
```

Comment:
Move this to a separate file Replicas.json

Reply:
Done